dessia_common/serialization.py

```diff
@@ -56,16 +57,20 @@ def _serializable_dict(self):
                  if k not in self._non_serializable_attributes and not k.startswith('_')}
         return dict_
 
-    def to_dict(self, use_pointers: bool = True, memo=None, path: str = '#') -> JsonSerializable:
-        """ Generic to_dict method. """
+    def to_dict(self, use_pointers: bool = True, memo=None, path: str = '#',
+                id_method=True, id_memo=None) -> JsonSerializable:
+        """
+        Generic to_dict method.
```

Comment:
Just as an indication, pydocstyle states that one line docstring shouldn't be set as three line (D200). More generally, I think we could activate D2xx errors